package/base-files/files/lib/functions/ipv4.sh

```diff
@@ -0,0 +1,142 @@
+#
+# rewrite of /bin/ipcalc.sh from awk to shell for use as a library
+# in init.d wrappers, and better portability (since busybox awk
+# lacks -E support at this time).
+#
```

Comment:
I'd say, the comment can go :-)

Once this PR is merged, `ipcalc.sh` will be a program using this library (not something this file is a rewrite of).
No need to remind people that this could be implemented using `awk -E`.

Reply:
Done